_integration_tests/asdf/install_nodejs_test.go

```diff
@@ -17,7 +16,7 @@ func TestAsdfInstallNodeVersion(t *testing.T) {
 		expectedVersion    string
 	}{
 		{"Install specific version", "18.16.0", provider.ResolutionStrategyStrict, "18.16.0"},
-		{"Install partial major version", "20", provider.ResolutionStrategyLatestInstalled, "20.19.3"},
```

Comment:
Node 20 is still actively maintained and `20.19.4` was released in the meantime -> let's switch to an EOL version in tests